packages/base/src/util/AriaLabelHelper.ts

```diff
@@ -1,7 +1,19 @@
+import type UI5Element from "../UI5Element.js";
+
+type InvalidateCallback = (param: any) => void;
+const observers = new WeakMap<HTMLElement, MutationObserver>();
+const inputAssociatedLabels = new WeakMap<UI5Element, Array<HTMLElement>>();
+const elementPropertyChangeCallback = new WeakMap<UI5Element, InvalidateCallback>();
```

Comment:
`elementPropertyChangeCallback` sounds alike a method but its' a map maybe `elementPropertyChangeCallbackMap`

Reply:
elementPropertyChangeCallbacks is more consistent with the rest